src/App.tsx

```diff
@@ -54,6 +61,32 @@ function App() {
             times.push(time3)
 
             setTimes(times.slice())
+
+            const [, time4] = await run(() => {
+                const member = new Identity().commitment
```

Comment:
We can remove this I think, and just add a number as the identity. Below too. So that the time only includes additions/updates.